package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "teacher-moments",
+  "version": "1.0.0",
+  "description": "Teacher Moments: Practice spaces for teacher preparation programs",
+  "license": "MIT",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "webpack-dev-server --mode development",
+    "build": "webpack --mode development",
+    "lint": "eslint src/**/*.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bocoup/threeflows.git"
+  },
+  "author": "Bocoup, LLC - GAIA team (https://bocoup.com)",
+  "bugs": {
+    "url": "https://github.com/bocoup/threeflows/issues"
+  },
+  "proxy": "localhost:3000/",
+  "dependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.6",
+    "copy-webpack-plugin": "^5.0.4",
+    "css-loader": "^3.2.0",
+    "eslint": "^6.2.1",
+    "eslint-plugin-react": "^7.14.3",
+    "express": "^4.17.1",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
+    "react-hot-loader": "^4.12.11",
+    "style-loader": "^1.0.0",
+    "webpack": "^4.39.2",
```

Comment:
are webpack and babel and react "dependencies" or are the "devDependencies" ?

Reply:
As discussed offline, almost all of these should be devDependencies. However we'll tackle that in a follow up PR to add Yarn workspaces